modules/core/src/passes/layers-pass.ts

```diff
@@ -191,7 +199,9 @@ export default class LayersPass extends Pass {
           moduleParameters
         );
         layerParam.layerParameters = {
+          ...DEFAULT_PARAMETERS,
           ...layer.context.deck?.props.parameters,
+          ...layer.props.parameters,
```

Comment:
Just an idea: would it be useful if `props.parameters` could also accept a function to allow the application to control the final parameters? By passing in the `Pass` the `Layer` could have more control:

```ts
new Layer({
  ...,
  parameters: (pass: Pass, parameters: Parameters): Parameters {...}

})
```